src/Fieldtypes/TemplateFolder.php

```diff
@@ -16,13 +16,6 @@ protected function toItemArray($id, $site = null)
 
     public function getIndexItems($request)
     {
-        return Folder::disk('resources')
-            ->getFoldersRecursively('views')
-            ->map(function ($folder) {
-                $folder = str_replace_first('views/', '', $folder);
-
-                return ['id' => $folder, 'title' => $folder];
-            })
-            ->values();
+        return Views::all();
```

Comment:
This is used when you edit a `template` field in the blueprint editor. It's the "Restrict to folder" config. The dropdown doesn't work at all because the ajax call is throwing an exception.

Even if the dropdown did work, it's supposed to be a list of subdirectories in the view directory. Now, it's attempting to just list all the views.

Reply:
Ah good point! I'll sort this on Friday - have made this PR a draft in the meantime.